timesketch/models/annotations.py

```diff
@@ -339,12 +360,21 @@ def set_status(self, status):
     def get_status(self):
         """Get the current status.
 
+        Only one status should be in the database at a time.
+
+        Raises:
+            RuntimeError: If more than one status is available.
+
         Returns:
             The status as a string
         """
         if not self.status:
             self.status.append(self.Status(user=None, status="new"))
-        return self.status[0]
+        if len(self.status) > 1:
+            # TODO: at some point replace the waning with an exception:
+            # raise RuntimeError("More than one status available")
+            logging.warning("More than one status available")
+        return self.status[0]  # always return the last element in the array
```

Comment:
Some minor changes to the wording and adding some data to make the warning and exception more debuggable for admins.

```suggestion
            self_id = self.id if hasattr(self, "id") else None
            # TODO: Change from warning to raising an exception once we ensured
            # it won't affect the deployment.
            # raise RuntimeError(
            # "More than one status available for object [%s] with ID: [%s]",
            #     str(type(self).__name__),
            #     str(self_id)
            # )
            logging.warning(
                "More than one status available for object [%s] with ID: [%s]",
                str(type(self).__name__),
                str(self_id)
            )
        return self.status[0]
```